src/project_workspace.ts

```diff
@@ -6,6 +6,16 @@
  *
  */
 
+export interface ProjectWorkspaceConfig {
+  jestCommandLine: string;
+  pathToConfig?: string;
```

Comment:
I hate to mention this now, but if we are going for jestCommandLine, do you think we should also retire `pathToConfig`? Sorry, I should have thought about this earlier...

Reply:
IMO I think the `pathToConfig` property is still needed. It just needs to be optional rather than required. Since I was introducing the `ProjectWorkspaceConfig` interface, which hopefully will be the future, I thought I'd mark `pathToConfig` as optional from the get go.

Reply:
I am not sure what the use case will be... but I am fine in leaving it in for now.